nixos/tests/end-to-end.nix

```diff
@@ -139,6 +139,10 @@ nixosTest {
 
     # transfer
     beneficiary = client.succeed("cat ${clientUsersDirectory}/user-3/public_key_hex")
+    # TODO we currently have to pass the nonce explicitly as we are lacking a way to synchronize
+    # the successful creation of an account in the account trie after a deposit.
+    # After adding the DA and awaiting that the server picked up the new deposit and thus
+    # an according account was added to the trie, we can discard the nonce and get it from the server
```

Comment:
Wait mechanism for deposit is necessary indeed, but it does not have to be done with DA.